pkg/network/ovn_kubernetes.go

```diff
@@ -222,9 +222,6 @@ func isOVNKubernetesChangeSafe(prev, next *operv1.NetworkSpec) []error {
 			errs = append(errs, errors.Errorf("cannot edit a running hybrid overlay network"))
 		}
 	}
-	if pn.IPsecConfig == nil && nn.IPsecConfig != nil {
-		errs = append(errs, errors.Errorf("cannot enable IPsec after install time"))
-	}
 	if pn.IPsecConfig != nil {
 		if !reflect.DeepEqual(pn.IPsecConfig, nn.IPsecConfig) {
```

Comment:
@markdgray wouldn't this block trigger and fail when disabling IPsec on the cluster? at least I'd that a DeepEqual() on `pn != nil` and `nn == nil` would return false, but maybe DeepEqual() holds untold surprises within itself...

Reply:
no, `next` (`nn`) would be nil and thus the conditional would skip.

Reply:
I meant to mark Dan's comment as resolved. I fixed this in the first iteration.